docs/examples/Asymmetric Line.ipynb.ipynb

```diff
@@ -0,0 +1,342 @@
+   "id": "682c1c48",
+   "metadata": {},
+   "source": [
+    "### One-time State Estimation\n",
+    "Below we present a simple example of state estimation for a network with a two asym lines. \n",
+    "\n",
+    "NOTE: In `power-grid-model`, asym lines belong to `branch` component type, therefore the `measured_terminal_type` of power sensors should be assigned to `MeasuredTerminalType.branch_from/_to`.\n",
+    "\n",
+    "For detailed explanation of the arguments, batch calculations and asymmetric calculations, we refer to the [State Estimation Example](./State%20Estimation%20Example.ipynb) and [Asymmetric Calculation Example](./Asymmetric%20Calculation%20Example.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "id": "f0c8c3e8",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ValueError",
+     "evalue": "setting an array element with a sequence. The requested array would exceed the maximum number of dimension of 1.",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
+      "\u001b[31mValueError\u001b[39m                                Traceback (most recent call last)",
+      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[40]\u001b[39m\u001b[32m, line 6\u001b[39m\n\u001b[32m      4\u001b[39m asym_voltage_sensor[\u001b[33m\"\u001b[39m\u001b[33mmeasured_object\u001b[39m\u001b[33m\"\u001b[39m] = [\u001b[32m2\u001b[39m, \u001b[32m6\u001b[39m]\n\u001b[32m      5\u001b[39m asym_voltage_sensor[\u001b[33m\"\u001b[39m\u001b[33mu_sigma\u001b[39m\u001b[33m\"\u001b[39m] = [\u001b[32m1.0\u001b[39m, \u001b[32m1.0\u001b[39m]\n\u001b[32m----> \u001b[39m\u001b[32m6\u001b[39m \u001b[43masym_voltage_sensor\u001b[49m\u001b[43m[\u001b[49m\u001b[33;43m\"\u001b[39;49m\u001b[33;43mu_measured\u001b[39;49m\u001b[33;43m\"\u001b[39;49m\u001b[43m]\u001b[49m = [[\u001b[32m5640\u001b[39m, \u001b[32m5000\u001b[39m, \u001b[32m6000\u001b[39m], [\u001b[32m5640\u001b[39m, \u001b[32m5000\u001b[39m, \u001b[32m6000\u001b[39m]]\n\u001b[32m      8\u001b[39m \u001b[38;5;66;03m# power sensor\u001b[39;00m\n\u001b[32m      9\u001b[39m asym_power_sensor = initialize_array(DatasetType.input, ComponentType.sym_power_sensor, \u001b[32m1\u001b[39m)\n",
+      "\u001b[31mValueError\u001b[39m: setting an array element with a sequence. The requested array would exceed the maximum number of dimension of 1."
+     ]
+    }
+   ],
+   "source": [
+    "# voltage sensor\n",
+    "asym_voltage_sensor = initialize_array(DatasetType.input, ComponentType.asym_voltage_sensor, 1)\n",
+    "asym_voltage_sensor[\"id\"] = [8]\n",
+    "asym_voltage_sensor[\"measured_object\"] = [6]\n",
+    "asym_voltage_sensor[\"u_sigma\"] = [1.0]\n",
+    "asym_voltage_sensor[\"u_measured\"] = [[5640, 5000, 6000]]\n",
+    "\n",
+    "# power sensor\n",
+    "asym_power_sensor = initialize_array(DatasetType.input, ComponentType.sym_power_sensor, 4)\n",
```

Comment:
```suggestion
    "asym_power_sensor = initialize_array(DatasetType.input, ComponentType.asym_power_sensor, 4)\n",
```
This and re-rendering of output. 
nitpick: Its nice to "restart kernal and run all so that the execution  count starts from 1.